jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard.js

```diff
@@ -605,7 +647,9 @@ define([
         this.$container.one('transitionend',
             containerTransitionEnd.resolve.bind(containerTransitionEnd));
         $.when(cellTransitionEnd, containerTransitionEnd)
-            .then(this._repositionHiddenCells.bind(this));
+            .then(function() {
+                this._repositionHiddenCells();
+            }.bind(this));
```

Comment:
That might be a leftover from refactoring. I think I had some other code in there at one point and then moved it out. I can fix it back to the original.